package.json

```diff
@@ -20,7 +20,7 @@
   },
   "bugs:": "https://github.com/dnsimple/dnsimple-node/issues",
   "engines": {
-    "node": ">= v6.0.0"
+    "node": ">= v8.0.0"
```

Comment:
As @weppos said I think we need a major release since we bump from minimum from v6 to v8.

All because of a dev dependency, `loadash`, that doesn’t run on v6 anymore, which makes more sense since v6 is deprecated.